README.md

```diff
@@ -97,8 +71,6 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
 - **[key_feature_1]**
 - **[key_feature_2]**
 - **[key_feature_3]**
```

Comment:
- [ ] Good work leaving the Key Features, kindly add 1-3 features that you implemented in this project. It would make it easier to understand what you built by reading the README.md file. 👍